CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Features
 
-- Report start up crashes (#2220)
+- Report [start up crashes](https://docs.sentry.io/platforms/apple/guides/ios/) (#2220)
```

Comment:
Is linking to a feature list really that useful? Isn't there a more specific page to link to?

Reply:
We don't have a page for that, and I don't think we will add one. It's one sentence. Would you like to see a more detailed explanation of the feature?

Reply:
In that case I am not sure the link adds any value at all, since it's a feature description linking to a feature description sentence within a rather big page. But no problem to merge it, it doesn't hurt either.

Reply:
Yeah, it's better than nothing.